src/blocks/card.css

```diff
@@ -0,0 +1,88 @@
+.card {
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+  justify-content: center;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  align-self: center;
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+}
+
+.card__title {
+  font-family: inherit;
+  font-size: inherit;
+  font-weight: 500;
+  line-height: 18px;
+  color: var(--main-color);
+  margin-bottom: 4px;
+}
+
+.card__code {
+  line-height: 14px;
+  font-size: 10px;
+  color: var(--secondary-color);
+  margin-bottom: 16px;
+}
+
+.card__rate {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-bottom: 24px;
+}
+
+.card__review {
+  line-height: 14px;
+  color: var(--main-color);
+  font-size: 10px;
+}
```

Comment:
the font-size and line-height are duplicated for the card__code and card__review. maybe it's better to take them to a separate class?

Reply:
You're probably right, but I don't think it makes sense to create a separate class for the two elements. We need to ask the mentor)